backend/Models/Database/WhaleSpecies.cs

```diff
@@ -0,0 +1,12 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class WhaleSpecies
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string ScientificName { get; set; }
+        public string Description { get; set; }
+        public string PhotoUrl { get; set; }
+        public string ConservationStatusId { get; set; }
+    }
```

Comment:
You need to add a list both here and in `Location.cs` to allow Entity Framework to set up the join table:
```suggestion
        public List<Location> Locations { get; set; }
    }
```